### PR TITLE
Cleaning duplicated procs, part 2

### PR DESCRIPTION
**Pull request by Zth--**
Continuation of #24682

I'm leaving the hard ones for the last part. 

~~33 duplicated procs left: https://pastebin.com/3z0Z0yTU~~
17 duplicated procs left: https://pastebin.com/eAYkz9Bd

### CODE REVIEW

**Comment by Irradiation** [comment]
Doing God's work.

**Comment by ghost** [comment]
dog bless

**Comment by Exxion** [comment]
Huh. They're not direct duplicates, but it looks like the `power_outage` and `power_disabled` events are identical

File: code/modules/bomberman/bomberman.dm [hunk 1]
```diff
@@ -473,10 +473,10 @@ obj/structure/bomberflame/Destroy()
 	icon_state = "hardwall"
 	opacity = 0
 
-/turf/unsimulated/wall/ex_act(severity)
```
Comment (ShiftyRail) [inline code/modules/bomberman/bomberman.dm]: Did you forget to remove the `return`s ?

**Pull request merged**